src/box/lua/config/applier/credentials.lua

```diff
@@ -559,16 +559,20 @@ local function sync_privileges(credentials, obj_to_sync)
                     obj_to_sync.type, obj_to_sync.name)
     end
 
-    -- Drop missed privilege alerts.
+    -- Prepare to drop missed privilege alerts.
     --
     -- The actual alerts will be issued in the sync() function
     -- below.
     --
     -- Important: we don't follow the `obj_to_sync` filter here
     -- for simplicity. Just revisit all the missed privilege
-    -- alerts: drop all of them and issue again the actual ones.
-    config._aboard:drop_if(function(_key, alert)
-        return alert._trait == 'missed_privilege'
+    -- alerts: mark all of them and issue again the actual ones.
```

Comment:
Nit:
> mark all of them **obsolete** and issue the actual once

Reply:
I'm not sure about this. If I write it like that, it will look like `obsolete` is an alert status, but that's just an internal trait. I think that the code is not complex enough to have more explanation, than this.

Reply:
The point is, that I can't really understand from this sentence, how you mark them. If you mark them obsolete or `missed_privilege_obsolete`, it is clearer.

Reply:
I am not sure that it is needed to have complete description of the code.

Reply:
As you wish.